app/services/breadcrumbs_options.rb

```diff
@@ -30,11 +30,24 @@ def index_item
     end
 
     {
-      caption: _(resource_name.pluralize.humanize),
-      url: resource_path(class_name)
+      caption: get_caption(resource_name),
+      url: resource_path(class_name) || resource_path(resource_name)
     }
   end
 
+  def get_caption(resource_name)
+    case resource_name
```

Comment:
@tbrisker, could you advise for a more generic solution here, if any?

Reply:
Could you extract this to a helper for a future reuse?